remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/CompatibilityProtoStreamMarshaller.java

```diff
@@ -21,6 +21,10 @@
    public CompatibilityProtoStreamMarshaller() {
    }
 
+   public CompatibilityProtoStreamMarshaller(EmbeddedCacheManager cacheManager) {
```

Comment:
Was wondering why we need the cacheManager as a constructor parameter and then also injected later. But it seems this constructor is not used anyway.